apps/users-service/src/cms/romps.ts

```diff
@@ -0,0 +1,43 @@
+import { Base, BaseOptions } from '@asap-hub/services-common';
+import { Romp } from '../entities/romps';
+
+export interface CreateRomp {
```

Comment:
We can throw this into `packages/model` already so it's accessible to FE+BE :upside_down_face: